ci/jenkins/templates/e2e-dev-template.yaml

```diff
@@ -35,8 +35,8 @@
         - string:
               name: REPO_BRANCH 
               default: ''
-              description: repository branch used for skuba packages
-    pipeline-scm:
+              description: Branch of the CaaSP repository used for packages and images
+     pipeline-scm:
```

Comment:
This one additional space breaks parsing, see CI,

Reply:
good catch, thanks!